t/14-grutasks.t

```diff
@@ -102,17 +105,6 @@ my $pid       = OpenQA::Test::Utils::create_webapi($mojo_port, sub { });
 # we possibly want to adjust without going into the details of this test
 $t->app->config->{default_group_limits}->{asset_size_limit} = 100;
 
-# Allow Devel::Cover to collect stats for background jobs
-$t->app->minion->on(
-    worker => sub {
-        my ($minion, $worker) = @_;
-        $worker->on(
-            dequeue => sub {
-                my ($worker, $job) = @_;
-                $job->on(cleanup => sub { Devel::Cover::report() if Devel::Cover->can('report') });
-            });
-    });
```

Comment:
You could enqueue the first job with a `note` and only set the cleanup event handler when the note is present. Then you only get coverage taken once.

Reply:
Or the other way around.

Reply:
done, PR updated.